src/ts/errorutils.ts

```diff
@@ -73,7 +73,7 @@ export namespace ErrorUtils {
   };
 
   export const defaultWebErrorMessageTranslator: ErrorMessageTranslator = (error: ErrorEvent) => {
-    const errorMessage = ErrorUtils.defaultErrorMessages[error.code];
+    const errorMessage = error.message ?? ErrorUtils.defaultErrorMessages[error.code];
```

Comment:
Have you checked with the mobile teams if their `error.message` property should be used here as well?

Reply:
The handling for that is in a separate function, this should be Web only I believe.

Reply:
true that 🤔 missed that 👍